### DISCUSSION TITLE
On what system do you use NovelWriter?

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#2125] [General]
Do you use NovelWriter on an overclocked water cooled Core i9 with 64GB of RAM, or do you use it with a lowly Ryzen 3 with 4GB of RAM?

I use it on an Asus MiniPC PN50. It has a lowly Ryzen 3 (4800U iirc) and 8GB of RAM. I am not tech-savvy enough to increase this.

**Comment by vkbo** [comment]
> Do you use NovelWriter on an overclocked water cooled Core i9 with 64GB of RAM

Well, actually, yes. I work on a water cooled 16 core i9-12900K with 64GB of DDR5 RAM. 😄 

I do test on other computers and in limited virtual machines though, so I hope novelWriter still runs smoothly?

**Reply by xahodo** [reply]
2.5 seems to work just fine. I have not tested 2.6 beta 1.
